### PR TITLE
🌐 Add i18n support

### LINKED ISSUE #96: Add i18n support
We should support at least English to make life for international students easier

### PR DESCRIPTION
**Pull request by BuildmodeOne**
Closes #96 

### Summary

Added internationalization support to various components of the app using `next-i18next` and the `useTranslation` hook. Created a `LanguageModal` component to allow users to switch languages. Translated card titles, texts, and UI elements using the `t` function and translation namespaces. Modified the `allergens.json` file to store allergen names in both German and English. Used the DeepL API to translate room functions.

> _Sing, O Muse, of the glorious pull request_
> _That brought to the app the gift of tongues_
> _And made it speak in German and in English_
> _With `useTranslation` and `t` functions._

### Walkthrough
*  Add `LanguageModal` component to allow users to change the language of the app (F15)
*  Add `NEXT_PUBLIC_DEEPL_ENDPOINT` environment variable to enable the use of the DeepL API for translating room functions ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-2e7ddefda45db8e5ef729a8474280d19367dc279f0864c2f041f9ed4f145e729R7))
*  Remove `label` property and add `i18nKey` property to the card objects in the `ALL_DASHBOARD_CARDS` array ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-af4ec176af3bf8a0f9b885f1a9abeda79272eed848468a7375071c713d8304e6L24)-[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-af4ec176af3bf8a0f9b885f1a9abeda79272eed848468a7375071c713d8304e6L127-R116))
*  Modify `BaseCard` component to accept `i18nKey` prop and use `t` function to display the translated card title ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-6d5a974bca88a22178396045fe06d42038a21a15261859ca154843c476b16297R11),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-6d5a974bca88a22178396045fe06d42038a21a15261859ca154843c476b16297L20-R22),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-6d5a974bca88a22178396045fe06d42038a21a15261859ca154843c476b16297L28-R32),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-6d5a974bca88a22178396045fe06d42038a21a15261859ca154843c476b16297L40-R47))
*  Modify `CalendarCard`, `EventsCard`, `FoodCard`, `RoomCard`, and `TimetableCard` components to pass `i18nKey` prop to `BaseCard` component ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-523cbfe2fd6aeaaadf4887be6f0adfc43dbc03c3a9e078617236fa508098fca7L49-R51),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fe999f8feb1da8a753a5523d1f04bce8db59b2d356ab784aecb30799bddccf11L34-R37),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495L75-R92),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-07f9adf638ab19414beffe94ac05448855d9215944059a115b69c7319a86a725L74-R76),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-b8830888ba52add5883328504e8e377eda8981480feebb61533060d08b1e1317L46-R48))
*  Modify `CalendarCard`, `ElectionPrompt`, `EventsCard`, `FoodCard`, `InstallPrompt`, `MobilityCard`, `RoomCard`, `SurveyPrompt`, `DashboardModal`, `FilterFoodModal`, `PersonalDataModal`, and `ThemeModal` components to use `useTranslation` hook and `t` function to translate the texts and labels ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-523cbfe2fd6aeaaadf4887be6f0adfc43dbc03c3a9e078617236fa508098fca7R13),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-523cbfe2fd6aeaaadf4887be6f0adfc43dbc03c3a9e078617236fa508098fca7R21),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-523cbfe2fd6aeaaadf4887be6f0adfc43dbc03c3a9e078617236fa508098fca7L60-R63),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d1341e435ef76ace37434a575b98711aa84d34796a48597cfd718afc874e9b29R10),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d1341e435ef76ace37434a575b98711aa84d34796a48597cfd718afc874e9b29R19),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d1341e435ef76ace37434a575b98711aa84d34796a48597cfd718afc874e9b29L24-R37),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fe999f8feb1da8a753a5523d1f04bce8db59b2d356ab784aecb30799bddccf11R11-R12),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fe999f8feb1da8a753a5523d1f04bce8db59b2d356ab784aecb30799bddccf11R19),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fe999f8feb1da8a753a5523d1f04bce8db59b2d356ab784aecb30799bddccf11L44-R47),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495L10-R10),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495R23),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495L29-R46),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495L55-R67),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495L63-R79),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-d2d911400db0a99d962bada913c2d8448f015ec712a59d1e4117b4e265e55495L92-R108),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-1833a6452e954bcf1ca2b5bbf7a0c0adcf29e74d8e7e09b67d9c0c2e39091067R10),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-1833a6452e954bcf1ca2b5bbf7a0c0adcf29e74d8e7e09b67d9c0c2e39091067L41-R79),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fa960008036e3b1a091353d948d4f7695a477208cff8ffc54bef4da8b095f24aL13-R16),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fa960008036e3b1a091353d948d4f7695a477208cff8ffc54bef4da8b095f24aR22),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fa960008036e3b1a091353d948d4f7695a477208cff8ffc54bef4da8b095f24aL44-R47),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fa960008036e3b1a091353d948d4f7695a477208cff8ffc54bef4da8b095f24aL61-R72),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-fa960008036e3b1a091353d948d4f7695a477208cff8ffc54bef4da8b095f24aL76-R87),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-07f9adf638ab19414beffe94ac05448855d9215944059a115b69c7319a86a725R17),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-07f9adf638ab19414beffe94ac05448855d9215944059a115b69c7319a86a725R26),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-07f9adf638ab19414beffe94ac05448855d9215944059a115b69c7319a86a725L88-R90),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-bf5c102c97dbded558d9b366ade7e07c0a1b2ffd239578f99c42b8ac2e3b2aeaR10),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-bf5c102c97dbded558d9b366ade7e07c0a1b2ffd239578f99c42b8ac2e3b2aeaR19-R20),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-bf5c102c97dbded558d9b366ade7e07c0a1b2ffd239578f99c42b8ac2e3b2aeaL26-R29),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-488e6699e69b575b91f20fdb340b94858f43a570d0202631375de5964faf1112R1),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-488e6699e69b575b91f20fdb340b94858f43a570d0202631375de5964faf1112L7-R9),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-488e6699e69b575b91f20fdb340b94858f43a570d0202631375de5964faf1112L27-R38),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-488e6699e69b575b91f20fdb340b94858f43a570d0202631375de5964faf1112L41-R56),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-488e6699e69b575b91f20fdb340b94858f43a570d0202631375de5964faf1112L61-R73),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-488e6699e69b575b91f20fdb340b94858f43a570d0202631375de5964faf1112L82-R85),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aR15-R16),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aR41-R42),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL44-R48),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL50-R54),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL57-R61),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL63-R67),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL69-R73),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL79-R83),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL85-R94),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL97-R101),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL103-R113),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL115-R119),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL127-R131),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL136-R140),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL155-R159),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-c65dda90f449b1b9dacf2cc84063ec36f1f041869bd9c082af1c459d19aea60aL164-R168),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-b98747db66b2124ef00d9330b7de4f49ab3a4e2e6d1e6d6724310f622e2b314fL9-R26),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-b98747db66b2124ef00d9330b7de4f49ab3a4e2e6d1e6d6724310f622e2b314fL27-R32),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-b98747db66b2124ef00d9330b7de4f49ab3a4e2e6d1e6d6724310f622e2b314fR43-R44),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-b98747db66b2124ef00d9330b7de4f49ab3a4e2e6d1e6d6724310f622e2b314fL47-R59),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-b98747db66b2124ef00d9330b7de4f49ab3a4e2e6d1e6d6724310f622e2b314fL61-R74),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-094d0cb75b4886e918a965ce05e5a209c5281256613ce033876049923631d3caL11-R17),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-094d0cb75b4886e918a965ce05e5a209c5281256613ce033876049923631d3caR27),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-094d0cb75b4886e918a965ce05e5a209c5281256613ce033876049923631d3caL36-R57),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-094d0cb75b4886e918a965ce05e5a209c5281256613ce033876049923631d3caL53-R89))
*  Modify `AppNavbar` component to use `useTranslation` hook and `t` function to translate the icon titles ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-2bbfa973cc75dcd8f64d7809ab3343162c94ad40f6ba8b2de5acb03362b757b3R16),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-2bbfa973cc75dcd8f64d7809ab3343162c94ad40f6ba8b2de5acb03362b757b3R25-R26),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-2bbfa973cc75dcd8f64d7809ab3343162c94ad40f6ba8b2de5acb03362b757b3L47-R50),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-2bbfa973cc75dcd8f64d7809ab3343162c94ad40f6ba8b2de5acb03362b757b3L89-R97))
*  Modify `RoomMap` component to use `useTranslation` hook, `t` function, and `getTranslatedFunction` function to translate the room functions, floor names, and texts ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL1-R1),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcR18),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL55-R64),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL94-R114),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL110-R126),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcR149-R163),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL168-R205),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL179-R211),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL219-R253),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL228-R260),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL236-R268),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL256-R288),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL268-R304),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL278-R310),[link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-792530fabbecd04fdc814d15979bcb77c6af8beba521aba75e882a8bb526cbdcL289-R321))
*  Modify `allergens.json` file to store the allergen names in both German and English, using the language code as the key ([link](https://github.com/neuland-ingolstadt/neuland.app/pull/288/files?diff=unified&w=0#diff-768d2855912940a23e529b0799b1058ffe28601d6e743e31197bd32bee789aebL2-R149))

### CODE REVIEW

**Comment by Robert27** [comment]
Issue #265 is still missing and must be implemented before merge / release.

**Comment by BuildmodeOne** [comment]
> Issue #265 is still missing and must be implemented before merge / release.

I just discussed this topic with @alexhorn and we came to the conclusion that a English privacy policy would not be necessary doe to the fact that most of the users are German.

Additionally, as far as im concerned the GDPR does not demand a English version.

**Comment by Robert27** [comment]
> > Issue #265 is still missing and must be implemented before merge / release.
> 
> I just discussed this topic with @alexhorn and we came to the conclusion that a English privacy policy would not be necessary doe to the fact that most of the users are German.
> 
> Additionally, as far as im concerned the GDPR does not demand a English version.

That's right, I probably mixed something up about GDPR. Although it is usually best practice and would certainly make sense in the long term.

File: rogue-thi-app/pages/library.js
Comment (FabianSimonGross) [inline rogue-thi-app/pages/library.js:1]: image
Something here seems to be broken :)

File: rogue-thi-app/pages/food.js
Comment (FabianSimonGross) [inline rogue-thi-app/pages/food.js:1]: image
Ehm is it me or what is wrong here?

File: rogue-thi-app/pages/grades.js
Comment (FabianSimonGross) [inline rogue-thi-app/pages/grades.js:1]: This seems to not be translated at all am I right?

File: rogue-thi-app/components/modal/DashboardModal.js
Comment (FabianSimonGross) [inline rogue-thi-app/components/modal/DashboardModal.js:1]: This seems to not be translated either?

File: rogue-thi-app/pages/library.js
[the remark this answers is not in the document]
Comment (FabianSimonGross) [reply rogue-thi-app/pages/library.js:1]: This still is there even in the newest version "at least on two of my machines"

File: rogue-thi-app/pages/food.js
Comment (FabianSimonGross) [reply rogue-thi-app/pages/food.js:1]: This seems to be worse now. You get a "Error retrievieving the menu. Something seems to be broken" and if you go there everything "goes to hell" kind of
grafik

**Comment by FabianSimonGross** [comment]
> @FabianSimonGross are you sure you are running the latest version? I double checked all pages you mentioned and have no problems whatsoever.

Yes somehow my sync failed now i checked it again with the newest version. Still some things are there to do 👯

**Comment by BuildmodeOne** [comment]
> > @FabianSimonGross are you sure you are running the latest version? I double checked all pages you mentioned and have no problems whatsoever.
> 
> Yes somehow my sync failed now i checked it again with the newest version. Still some things are there to do 👯 

Okay, some things won't work because you need to set a DeepL API key in your .env.local file.

**Comment by BuildmodeOne** [comment]
@FabianSimonGross I just committed changes to use a simple fallback if DeepL is not configured. Can send you the DeepL API Key for real testing.

**Comment by BuildmodeOne** [comment]
@FabianSimonGross I just committed changes to use a simple fallback if DeepL is not configured. I can send you the DeepL API Key for real testing.

**Comment by FabianSimonGross** [comment]
Okay execpt for the Library Stuff it is working fine and should be :ok:

**Review by FabianSimonGross — APPROVED** [review]
Now it seems to work. I suggest merging.

Tested quite thoroughly by using it in german and english.

**Review by Robert27 — APPROVED** [review]

**Pull request merged**